content/blog/nats-operator-and-service-accounts.md

```diff
@@ -0,0 +1,337 @@
++++
+categories = ["engineering"]
+date = "2018-07-19"
+tags = ["kubernetes", "operators"]
+title = "NATS Operator: Integrating Kubernetes ServiceAccounts for operated NATS clusters"
+author = "Waldemar Quevedo"
++++
+
+The latest release of the [NATS Operator](https://github.com/nats-io/nats-operator) has alpha support for
+configuring via [CustomResourceDefinitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) the authorization rules for
+an operated NATS cluster by using Kubernetes [ServiceAccounts](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/) present
+in a namespace.
+
+In this blog post, we will explain how to use this features and share some details of
+the internals of how it currently works.
+
```

Comment:
In this blog post, we will explain how to use this feature along with details of the internals on how it currently works.